saleor/graphql/order/mutations/order_note_add.py

```diff
@@ -1,65 +1,40 @@
 import graphene
-from django.core.exceptions import ValidationError
 
 from ....core.tracing import traced_atomic_transaction
 from ....order import events
-from ....order.error_codes import OrderErrorCode
 from ....permission.enums import OrderPermissions
 from ...app.dataloaders import get_app_promise
 from ...core import ResolveInfo
+from ...core.descriptions import (
+    ADDED_IN_314,
+    DEPRECATED_IN_3X_INPUT,
+    DEPRECATED_IN_3X_MUTATION,
+    PREVIEW_FEATURE,
+)
 from ...core.doc_category import DOC_CATEGORY_ORDERS
-from ...core.mutations import BaseMutation
 from ...core.types import BaseInputObjectType, OrderError
-from ...core.validators import validate_required_string_field
 from ...plugins.dataloaders import get_plugin_manager_promise
 from ..types import Order, OrderEvent
+from .order_note_common import OrderNoteCommon
 from .utils import get_webhook_handler_by_order_status
 
 
-class OrderAddNoteInput(BaseInputObjectType):
-    message = graphene.String(
-        description="Note message.", name="message", required=True
-    )
-
-    class Meta:
-        doc_category = DOC_CATEGORY_ORDERS
-
-
-class OrderAddNote(BaseMutation):
+class OrderNoteAdd(OrderNoteCommon):
     order = graphene.Field(Order, description="Order with the note added.")
     event = graphene.Field(OrderEvent, description="Order note created.")
 
-    class Arguments:
+    class Arguments(OrderNoteCommon.Arguments):
         id = graphene.ID(
             required=True,
             description="ID of the order to add a note for.",
             name="order",
         )
-        input = OrderAddNoteInput(
-            required=True, description="Fields required to create a note for the order."
-        )
 
     class Meta:
-        description = "Adds note to the order."
+        description = "Adds note to the order." + ADDED_IN_314 + PREVIEW_FEATURE
```

Comment:
I think the scope of 3.14 is closed. We could release it in 3.15. 
```suggestion
        description = "Adds note to the order." + ADDED_IN_315 + PREVIEW_FEATURE
```